packages/components-css/skip-link-css/src/_mixin.scss

```diff
@@ -1,3 +1,18 @@
-/* stylelint-disable-next-line block-no-empty */
 @mixin nl-skip-link {
+  background-color: var(--nl-skip-link-background-color, Canvas);
+  color: var(--nl-skip-link-color, LinkText);
+  inset-inline-start: -100%;
+  min-block-size: var(--nl-skip-link-min-block-size, 44px);
+  min-inline-size: var(--nl-skip-link-min-inline-size, 44px);
+  padding-block: var(--nl-skip-link-padding-block);
+  padding-inline: var(--nl-skip-link-padding-inline);
+  position: fixed;
+  text-decoration: var(--nl-skip-link-text-decoration);
+}
+
+@mixin nl-skip-link--focus-visible {
+  background-color: var(--nl-skip-link-focus-visible-background-color);
+  color: var(--nl-skip-link-focus-visible-color);
+  position: static;
+  text-decoration: var(--nl-skip-link-focus-visible-text-decoration);
```

Comment:
Iets voor op de backlog: conistente focus-ring voor alle focusable componenten, en dan niet de default focus ring.

Reply:
Is dat iets voor common tokens of juist een eigen (CSS-only?) component? Beide?

Reply:
Ik denk dat juist de focus stijl er een is die je niet als common token wil omdat het een toegankelijkheidseis is dat die focus stijl consistent is. Met de common token oplossing inclusief component tokens lijk je nog keuzes te hebben als artistic designer 🧑‍🎨 maar dat zou voor focus stijl echt niet moeten.